android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -1370,8 +1369,11 @@ public void setMyLocationTrackingMode(int myLocationTrackingMode) {
       return;
```

Comment:
If setting the same tracking mode the issue will still exist, right?

Reply:
moved `updateMyLocationEnabled` up